src/AspNetCoreModuleV2/OutOfProcessRequestHandler/dllmain.cxx

```diff
@@ -214,6 +214,12 @@ EnsureOutOfProcessInitializtion()
             goto Finished;
         }
 
+        hr = ALLOC_CACHE_HANDLER::StaticInitialize();
```

Comment:
FINISHED_IF_FAILED()

Reply:
I can change  it now. We need rescan the files to switch all of them.